backend/Dockerfile

```diff
@@ -1,10 +1,11 @@
-FROM tiangolo/uvicorn-gunicorn-fastapi:python3.8-slim
+FROM python:3.8-slim-buster
```

Comment:
Could you explain what was the issue here?

Reply:
The image from `tiangolo` runs the `uvicorn` command without the parameter we need (`loop=asyncio`).
Since we added an async loop to ensure that only one request can access the database at a given time, we need the loop to be async to prevent the error:
> RuntimeError: Task <Task pending name='Task-969' coro=<RequestResponseCycle.run_asgi() running at /usr/local/lib/python3.8/site-packages/uvicorn/protocols/http/httptools_impl.py:375> cb=[set.discard()]> got Future <Future pending> attached to a different loop